big_tests/tests/mod_global_distrib_SUITE.erl

```diff
@@ -909,9 +942,8 @@ refresh_node(NodeName, Config) ->
 
 connect_from_spec(UserSpec, Config) ->
     {ok, User} = escalus_client:start(Config, UserSpec, <<"res1">>),
-    escalus_story:send_initial_presence(User),
-    escalus:wait_for_stanza(User),
```

Comment:
Why don't we wait for the presence reply anymore?

Reply:
Because in global distribution they can arrive out of sync and makes our tests a lot more complicated to take that into consideration, when in the tests that are using `connect_from_spec` they're not relevant anyway.
`wait_for_stanza` was actually added recently when these tests were attempted to be fixed before (https://github.com/esl/MongooseIM/pull/2392/files) as a mean to sync a bit the processes, but in the end it didn't bring any improvements, as this adds a bit of sync for `ungraceful` but makes `graceful` a lot more likely to fail. So I decided to get rid of it altogether and focus the tests on what they're trying to test, the messages.

Reply:
Are you sure about this? Why would this arrive out of sync, since we're waiting as the same user, as the one who sent the initial presence in the first place?

Reply:
https://mongooseim.readthedocs.io/en/latest/modules/mod_global_distrib/#message-routing and as an example http://mongooseim-ct-results.s3-eu-west-1.amazonaws.com/branch/CI/mod_global_distrib/7057/internal_mnesia.22.0/big/ct_run.test@travis-job-5a57dae3-a595-4f45-a4eb-e4b0300aaafe.2019-08-26_10.02.52/big_tests.tests.mod_global_distrib_SUITE.logs/run.2019-08-26_10.02.54/mod_global_distrib_suite.test_pm_with_graceful_reconnection_to_different_server.67.html
It's all in the routing, tables might not be fresh and messages can be routed in circles until they arrive.